CHANGELOG.latest.md

```diff
@@ -1,3 +1,14 @@
-### Bugfixes
-* [EXTERNAL] Make the setProxyURL a promise to wait for the native code to set the proxy URL (#1033) contributed by @dangilbert (#1035) via Toni Rico (@tonidero)
-* Setting `proxyURL` should not require configure (#1032) via Cesar de la Vega (@vegaro)
+Welcome to version 8.0 of the React Native RevenueCat SDK! A full migration guide can be found [here](https://github.com/RevenueCat/react-native-purchases/blob/66d77607d1ce2f11dcb4954eac8cd5756d0ec211/migrations/v8-MIGRATION.md).
+
+### Breaking Changes
+
+- v8.0 Release (#1050) via Will Taylor (@fire-at-will)
```

Comment:
Might be good to add some more details in the changelog files about the breaking changes, and probably a reference to the migration guide. 

We can do this in a followup PR and update the release notes as well